omega/omega-connector/omega-connector-grpc/src/test/java/org/apache/servicecomb/saga/omega/connector/grpc/LoadBalancedClusterMessageSenderTest.java

```diff
@@ -291,10 +291,12 @@ private MyTxEventService(Queue<String> connected, Queue<TxEvent> events, int del
     public void onConnected(GrpcServiceConfig request, StreamObserver<GrpcCompensateCommand> responseObserver) {
       this.responseObserver = responseObserver;
       connected.add("Connected " + request.getServiceName());
+      System.out.println("Connected " + request.getServiceName());
```

Comment:
Please don't use the System.out.println in the test.

Reply:
This is just for debugging. The tests pass on my local machine but the CI report the failure.